src/sdp/Utils.cpp

```diff
@@ -291,7 +291,7 @@ namespace mediasoupclient
 						return false;
 
 					return (
-					  jsonAttributeIt->get<std::string>() == "cname" && jsonIdIt->get<uint32_t>() == firstSsrc);
+					  jsonAttributeIt->get<std::string>() == "cname" /*&& jsonIdIt->get<uint32_t>() == firstSsrc*/);
```

Comment:
Why this?

Reply:
I refer to project mediasoup-client, 
`const ssrcCnameLine = offerMediaObject.ssrcs
		.find((line: any) => line.attribute === 'cname');

	if (!ssrcCnameLine)
		throw new Error('a=ssrc line with cname information not found');`

Reply:
Ok, then please remove the commented code. @jmillan do you know why we were doing this different here than in mediasoup-client? It doesn't make any sense to me.

Reply:
removed🙂